mne_features/univariate.py

```diff
@@ -975,6 +978,51 @@ def compute_svd_entropy(data, tau=2, emb=10):
     return -np.sum(np.multiply(sv_norm, np.log2(sv_norm)), axis=-1)
 
 
+
+def compute_powercurve_deviation(data, sfreq, freq_range=[0.1, 50.]):
+    """ Linear fit to the log-log frequency-curve with fit error (per channel).
```

Comment:
About the docstring:

- a blank line is missing after the first line (summary line) of the docstring.
- `sfreq` should be first, `data` after. When `get_univariate_funcs(sfreq)` is called, `_get_feature_funcs` will transform `compute_powercurve_deviation` into a partial function where its first argument is given the value `sfreq` (this is what the line `feature_funcs[alias] = partial(func, sfreq)` does).
- the docstring should give the alias of the feature function and, if possible, references.
- the output shape must be 1D, therefore `(n_channels * 4,)` instead of `(n_channels, 4)`. Return `fit_info.ravel()`.
- a blank line is missing before the section `Returns`.